vault/raft.go

```diff
@@ -171,6 +171,26 @@ func (c *Core) setupRaftActiveNode(ctx context.Context) error {
 	}
 
 	c.logger.Info("starting raft active node")
+
+	// Read autopilot configuration from storage
+	autopilotConfig, err := c.autopilotConfiguration(ctx)
+	if err != nil {
+		return err
+	}
+	// If the config wasn't found in storage, then either the default config, or
+	// the HCL config would have been used to instantiate autopilot. The
+	// resulting configuration should be persisted in storage.
```

Comment:
Why do we need to persist the default config? Can we only persist if the config has been changed from default? Doing so would give us the flexibility to update the default config overtime and have it apply to existing clusters

Reply:
Good idea. Done.